localstack/testing/pytest/fixtures.py

```diff
@@ -656,7 +656,7 @@ def is_stream_ready():
                 "UPDATING",
             ]
 
```

Comment:
idea: The triple nesting can be confusing to use. A usage example would be helpful.

Reply:
You are right but that is pretty unrelated to the PR. Would definitely be nice to add more docs to the test fixtures though!